pages/home.md

```diff
@@ -88,14 +86,14 @@ permalink: /
           </div>
           <div class="column is-offset-1 is-5">
             <h2 class="title">
-              Using interoperable data standards
+              Linked Data: Interoperable data standards
             </h2>
             <p class="content is-large">
             Linked Data gives Solid 
-            a common way to describe things and 
-            how they relate to each other, in a way that other people and 
-            machines can understand. This means that the data stored by Solid
-            is portable and completely interoperable.
+            a <strong>common</strong> way to describe things and 
+            their relationships to each other such that other people and 
+            applications can understand. This means that the data stored by Solid
```

Comment:
Can we reverse the sentence structure, with first the goal and then the means?
Because most won't care about Linked Data; they care about what we achieve with it.

> Solid ensures that _different_ apps can work with the _same_ data.

Reply:
Sure.  If amenable, could also take out the term "Linked Data"

Reply:
Updated to take out Linked Data for now.  Can, however, add a sentence For more information, see Linked Data (and link out).